app/views/fae/application/_credit.slim

```diff
@@ -1,5 +1,7 @@
 .credit
-  span a <a href="https://wearefine.com" target="_blank">FINE</a> Admin v#{Fae::Version.number}
+  span
+    | <a href="https://wearefine.com" target="_blank">FINE</a>
```

Comment:
This changes the markup. It prints as "a FINE Admin v###". This prints as "FINEAdmin v###"

If you guys are changing the markup to this, please Slimify this link - `a> href="https://wearefine.com" target="_blank" FINE`